src/service/error/AbstractFileError.ts

```diff
@@ -0,0 +1,10 @@
+export abstract class AbstractFileError extends Error {
+    constructor(className: string,
+                fileId: string,
+                messageProd: (fileId: string) => string,
```

Comment:
May I ask you to use full name to avoid confusion. I suspect you wanted this to be message supplier or producer.